EIPS/eip-3135.md

```diff
@@ -0,0 +1,203 @@
+---
+eip: 3135
+title: Exclusive Claimable Token
+author: Zhenyu Sun (@Ungigdu)
+discussions-to: https://github.com/ethereum/EIPs/issues/3132
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-8-10
+requires: 20
+---
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+
+This standard defines a token which can be claimed only by token issuer with payer's signature.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+
+When monitoring network traffic, counting paid e-mail messages or buy merchandise in general store, small but frequent payments are needed. We can't directly transfer tokens on chain to make payment because the sum gas cost is big and it takes time for blockchain to reach a consensus. 
+
+This standard allows service provider to establish micropayment channels with many users by creating claimable token. A user gets claimable token and can take part of balance as deposit to this channel. When using service, user should sign messages off chain to service provider and provider checks if user is honest with the accumulating credit in message and checks if this amount is less than deposit balance. If check passed, provider continue to serve user, otherwise reject. 
+
+At anytime, the service provider can claim the right amount of token on chain providing the last signed message it accepts. And this claim function can only be called by issuer to avoid double spending, that's the "exclusive" part of token.
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+There are two motivations for this standard, one is to reduce gas costs, the other is to link Ethereum to real-world payment problem: pay for usage.
+
+This token standard is useful in basically two types of businesses: 1. Real-world merchant (whose identity is known by user) can use this token as recharge card; 2. Online service provider (whose identity may not be known by user or is hard to reach) can use this token to build an automatic micropayment channel. In both cases, the token is the representation of provider's service/products, explained by provider and claimed by provider.
```

Comment:
The motivation section should explain why this needs to be a Standard rather than just a Good Idea.  It seems to me that every dapp can use a slightly different payment system and that is OK as they'll all be building their own UI anyway.  What interoperability benefits do we gain by having a universal standard for payment channels?

Reply:
Thank you for pointing this out. Well, for online service which needs to sign micropayment messages frequently, client should have a open wallet for signing. And payment systems are slightly different, as you say, it's not necessary to make it a standard. But for real-world businesses, like a coffee shop, payment signing only happens when checking out. A wallet like Imtoken or metamask is capable to do both deposit and signning, and the pay system is just the same. Many small businesses do not have the resourece to develop their own Dapp. So I think it may be useful to make it a standard.